release_notes.md

```diff
@@ -4,10 +4,6 @@
 - My change description (#PR)
 -->
 - Update Java Worker Version to [2.12.2](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.12.2)
-- Update Python Worker Version to [4.17.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.17.0)
-- Increased maximum HTTP request content size to 210000000 Bytes (~200MB)
-- Update Node.js Worker Version to [3.8.1](https://github.com/Azure/azure-functions-nodejs-worker/releases/tag/v3.8.1)
```

Comment:
Can you help us understand why the release notes are being updated during the release prep? I would expect the notes to be updated by the change PRs and reset on new releases.

Reply:
Hi Fabio

As part of the process for create a Host prep PR we actually following below steps
1) Updating the Host version 
2) we are resetting the release notes on this PR and removing release notes that were in previous version 4.27.0